### PR TITLE
Add session refresh interval in Cassandra connector

### LINKED ISSUE #4838: Cassandra connector doesn't discover new keyspaces/tables with Yugabyte
I'm using the Cassandra connector to talk to Yugabyte's YCQL endpoint. I notice that if I add new keyspaces/tables through Yugabyte, I can't see them in Presto unless I restart Presto. This means "SHOW SCHEMAS" and "SHOW TABLES" don't reflect the new keyspaces/tables, but also I can't "SELECT" or otherwise query from the new tables.

### PR DESCRIPTION
**Pull request by ebyhr**
Fixes #4838

I considered extending BaseConnectorTest and BaseConnectorSmokeTest for Yugabyte tests at first, but there were some failures due to lack of `comment` table properties in Yugabyte. Cassandra connector uses `comment` to mange hidden columns. 

Also, excluded the logic to populate TPCH data in Yugabyte query runner because it was flaky, returning the wrong row number just after insert.

### CODE REVIEW

File: docs/src/main/sphinx/connector/cassandra.rst [hunk 1]
```diff
@@ -104,6 +104,9 @@ Property Name                                                 Description
 
 ``cassandra.batch-size``                                      Maximum number of statements to execute in one batch.
 
+``cassandra.session-refresh-interval``                        Refresh interval for initializing a session. New keyspaces or tables
```
Comment (hashhar) [inline docs/src/main/sphinx/connector/cassandra.rst:107]: Would `metadata` instead of `session` be clearer?

**Review by hashhar — APPROVED** [review]
Looks good. Is this required for https://github.com/trinodb/trino/pull/5708?

**Review by hashhar — APPROVED** [review]
Can you mention in 2nd commit's message that it's to make it work with Yugabyte CQL tables?

Also, some recommendation in docs to enable set a session refresh interval when connecting to Yugabyte would be helpful.

LGTM otherwise.

**Comment by ebyhr** [comment]
@findinpath Can you test if we need this config properties even after #7828?

**Comment by findinpath** [comment]
@ebyhr I rebased your changes on top of  the cassandra v4 changes and created a new branch
https://github.com/findinpath/trino/tree/cassandra-v4-yugabyte

It doesn't quite work out of the box.


`com.datastax.oss.driver.api.core.session.Session#getMetadata` states 

> Do not call it once and store the result, because it is a frozen snapshot that will become stale over time.

After doing the change:
`session.refreshSchema().getKeyspaces()`  in `io.trino.plugin.cassandra.CassandraSession#getCaseSensitiveSchemaNames` 
there is another exception thrown:

```
Caused by: java.lang.IllegalStateException: Can't find referenced user type jsonb
	at com.datastax.oss.driver.internal.core.metadata.schema.parsing.DataTypeCqlNameParser.parse(DataTypeCqlNameParser.java:84)
	at com.datastax.oss.driver.internal.core.metadata.schema.parsing.DataTypeCqlNameParser.parse(DataTypeCqlNameParser.java:51)
	at com.datastax.oss.driver.internal.core.metadata.schema.parsing.TableParser.parseTable(TableParser.java:170)
	at com.datastax.oss.driver.internal.core.metadata.schema.parsing.CassandraSchemaParser.parseTables(CassandraSchemaParser.java:178)
	at com.datastax.oss.driver.internal.core.metadata.schema.parsing.CassandraSchemaParser.parseKeyspace(CassandraSchemaParser.java:129)
	at com.datastax.oss.driver.internal.core.metadata.schema.parsing.CassandraSchemaParser.parse(CassandraSchemaParser.java:75)
```

**Comment by ebyhr** [comment]
@findinpath Thanks! 

Let me close this PR because it would be better to add the dedicated connector for Yugabyte. This change might be still required when adding the connector, but I wouldn't merge at this time.

**Pull request closed without merging**